boa/src/exec/break_node/mod.rs

```diff
@@ -0,0 +1,12 @@
+use super::{Executable, Interpreter};
+use crate::{
+    builtins::value::{ResultValue, Value},
+    syntax::ast::node::Break,
+};
+
+impl Executable for Break {
+    fn run(&self, interpreter: &mut Interpreter) -> ResultValue {
+        interpreter.is_break = true;
```

Comment:
Here, we could have a `break(label: &str)` function (signature might differ) in `interpreter`, which would change the state.